linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/tests/pam_tally2_absent_account_config.fail.sh

```diff
@@ -7,10 +7,24 @@ account	requisite			pam_deny.so
 account	required			pam_permit.so
 CAPTA
 
+{{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to sle products. So, why the conditionals here? Would be simpler to only update the file.